tests/config_settings/transition/py_args_tests.bzl

```diff
@@ -0,0 +1,68 @@
+# Copyright 2023 The Bazel Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+""
+
+load("@rules_testing//lib:test_suite.bzl", "test_suite")
+load("//python/config_settings/private:py_args.bzl", "py_args")  # buildifier: disable=bzl-visibility
+
+_tests = []
+
+def _test_py_args_default(env):
+    actual = py_args("foo", {})
+
+    want = {
+        "args": None,
+        "data": None,
+        "deps": None,
+        "env": None,
+        "main": "foo",
```

Comment:
Main should be a .py file, i.e. name + ".py" -- please fix.

I'm surprised this didn't didn't cause a failure somewhere. When given an explicit main, the py_binary should have failed when it tried to look for a non-existent file with the same name as itself